commit 5485ed31f513a515e95c7737347975a3148c3063 (refs/changes/10/3210/2)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-01-18T11:24:57-08:00 (1 year, 9 months ago)
    
    CNAPI-734 updating sysinfo is missing an argument
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/ur.js b/lib/ur.js
index 7716b54..8891540 100644
--- a/lib/ur.js
+++ b/lib/ur.js
@@ -87,7 +87,8 @@ Ur.prototype.serverSysinfo = function (uuid, opts, callback) {
         'timeoutSeconds': opts.timeoutSeconds,
         'message': {
             'type': 'file',
-            'file': '/usr/bin/sysinfo'
+            'file': '/usr/bin/sysinfo',
+            'args': ['-f']
         }
     };
     opts.uuid = uuid;
